zmon_worker_monitor/builtins/plugins/history.worker_plugin

```diff
@@ -18,8 +18,4 @@ Description = Builtin plugin that provides an http request check function.
 ;; Note valueX will be overridden if Zmon's global config has: plugin.{plugin_name}.{key} = valueY
 
 
-kairosdb_host = cassandra01
-
-kairosdb_port = 37629
-
-kairosdb_history_enabled = False
+;; url = cassandra01
```

Comment:
I left the ``url`` as indication for what config variables the plugin needs. I am ok with removing it, since it is already defined in ``config.yaml``.